delete-review-app/action.yml

```diff
@@ -43,6 +43,10 @@ inputs:
   tf-state-file:
     description: name of file containing terraform state
     required: true
+  tf-file:
```

Comment:
could you update the README.md for the action please. Looks a bit out of date so may need others adding too.